static/scripts/loggedin.js

```diff
@@ -66,42 +66,30 @@ $(document).ready(function () {
         $.showNotification(message, "success", true);
     }
 
-    /**
-     * creates the feedback-message which will be sent to the Schul-Cloud helpdesk
-     * @param modal {object} - modal containing content from feedback-form
-     */
-    const createFeedbackMessage = function (modal) {
-        return "Als " + modal.find('#role').val() + "\n" +
-            "möchte ich " + modal.find('#desire').val() + ",\n" +
-            "um " + modal.find("#benefit").val() + ".\n" +
-            "Akzeptanzkriterien: " + modal.find("#acceptance_criteria").val();
-    };
-
     const sendFeedback = function (modal, e) {
         e.preventDefault();
 
         let type = (modal[0].className.includes('feedback-modal')) ? 'feedback' : 'problem';
 
         let email = 'ticketsystem@schul-cloud.org';
         let subject = (type === 'feedback') ? 'Feedback' : 'Problem ' + modal.find('#title').val();
-        let text = createFeedbackMessage(modal);
-        let content = {text: text};
         let category = modal.find('#category').val();
-        let currentState = modal.find('#hasHappened').val();
-        let targetState = modal.find('#supposedToHappen').val();
 
         $.ajax({
             url: '/helpdesk',
             type: 'POST',
             data: {
-                email: email,
-                modalEmail: modal.find('#email').val(),
-                subject: subject,
-                content: content,
                 type: type,
+                subject: subject,
                 category: category,
-                currentState: currentState,
-                targetState: targetState
+                role: modal.find('#role').val(),
+                desire: modal.find('#desire').val(),
+                benefit: modal.find("#benefit").val(),
+                acceptanceCriteria: modal.find("#acceptance_criteria").val(),
+                currentState: modal.find('#hasHappened').val(),
+                targetState: modal.find('#supposedToHappen').val(),
+                email: email,
+                modalEmail: modal.find('#email').val(),
             },
             success: function (result) {
                 showAJAXSuccess("Feedback erfolgreich versendet!", modal);
```

Comment:
Couldn't this be more flexible? "Wunsch an das HPI"/"Problem an das HPI"/"Problem an die Schuladministration"
showAJAXSuccess(switch(type)...)

Reply:
My opinion: this is irrelevant. Nobody cares what exactly the success-banner says. Also, type only tells you whether the feedback is for the admin or hpi; type does not include information about whether it's an userstory or a problem. So there would be some more complex if clauses to achieve something quite unnecessary